### ISSUE TITLE
Possibilty for LM/GTE/PTE to add notes to translation memory.

### ISSUE THREAD

**Issue by NekoJonez** [GlotPress/GlotPress-WP#1719]
I'd love to see if LM/GTE/PTE can add notes to translation memory entries. This could be extremely helpful for those words that have multiple meanings.

Eg, we have to translate the string "Post". Without context (and especially if you cant read code or cant figure it out from the code) ... this can be a noun or a verb. 
So, the TM can have for Dutch: "Bericht" for noun and "Plaatsen"/"Publiceren" for verb

It might help new translators or CLPTE's when they use the TM to use the right translation for the context. And also us of course.

**Comment by amieiro** [comment]
You have the glossary to do this, because this information should be in the original string, related with your locale. If you go to https://translate.wordpress.org/locale/nl-be/default/glossary/, and you modify the 4th column for the "Post" term, adding these comments, when a nl-be translator will see the "Post" word in an original string, she will see this comment.

**Comment by NekoJonez** [comment]
Thats true, but it doesnt show in TM... why is one 100% different... you get me?
